.travis.yml

```diff
@@ -42,6 +43,7 @@ addons:
 
 env:
   global:
+    - DATABASE_URL=postgres://postgres:@localhost/cargo_registry_test
     - TEST_DATABASE_URL=postgres://postgres:@localhost/cargo_registry_test
```

Comment:
You mean just for `diesel database setup`, not migrations, right? because it looks like  `src/tests/all.rs` takes care of the migrations and uses TEST_DATABASE_URL? I just want to make sure I understand :)